transient/alt_threading.py

```diff
@@ -0,0 +1,22 @@
+import threading
```

Comment:
I'd rather this `alt_threading` module be called something like `checked_threading`

Reply:
Fixed in my most recent commit.